extensions/tyme-3-time-tracker/src/start-task-tracking.tsx

```diff
@@ -1,7 +1,11 @@
-import { Action, ActionPanel, Icon, List, closeMainWindow, showHUD } from "@raycast/api";
-import { Task, getTasks, startTrackingTask } from "./tasks";
+import { Action, ActionPanel, Icon, List, closeMainWindow, showHUD, getPreferenceValues } from "@raycast/api";
+import { Task, getTasks, startTrackingTask, stopTracking } from "./tasks";
 import { useCachedPromise, useFrecencySorting } from "@raycast/utils";
 
+interface Preferences {
+  stopRunningTimers: boolean;
```

Comment:
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`

Reply:
Hey, thanks for the review. I believe it's implemented properly now.